### PR TITLE
Import client nodejs

### PR DESCRIPTION
**Pull request by vmax**

### CODE REVIEW

File: .bazelrc [hunk 1]
```diff
@@ -0,0 +1,35 @@
+build:rbe --project_id=project-id
```
Comment (lolski) [inline .bazelrc]: please update project_id to `grakn-dev`

File: .bazelrc [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+build:rbe --remote_instance_name=projects/project-id/instances/default_instance
```
Comment (lolski) [inline .bazelrc]: adn this to `projects/grakn-dev/instances/default_instance`

File: .bazelrc (hunk 1 above)
Comment (vmax) [reply .bazelrc]: @lolski fixed in https://github.com/graknlabs/client-nodejs/commit/5abc8afe3145234c617c1f5aecc7c42911a18a7b

File: .bazelrc (hunk 2 above)
Comment (vmax) [reply .bazelrc]: @lolski fixed in https://github.com/graknlabs/client-nodejs/commit/5abc8afe3145234c617c1f5aecc7c42911a18a7b

**Pull request merged**